suite-common/wallet-core/src/send/sendFormSelectors.ts

```diff
@@ -58,7 +58,9 @@ export const selectSendFormReviewButtonRequestsCount = (
             code === 'ButtonRequest_SignTx' ||
             isCardano ||
             (isEthereum && code === 'ButtonRequest_Other') ||
-            (isStellar && code === 'ButtonRequest_Other'),
+            // This is a special case for T1B1 devices (Stellar).
+            // See https://github.com/trezor/trezor-firmware/issues/5120
+            (isStellar && (code === 'ButtonRequest_Other' || code === 'ButtonRequest_ProtectCall')),
```

Comment:
with this code, I am getting passphrase window before signing on non-passphrase wallet


https://github.com/user-attachments/assets/a323a8fe-df3b-4ea3-b284-9bca0d9bae65

Reply:
This issue doesn't seem to have been introduced in this PR; let me debug and see where the root cause is.

Reply:
Fixed in c555d57ca7943a12a2d16fc484720a004956dca7

🥲 I overlooked something, which led to this problem.